daily_notebook/20240701__modelling_bat_source_pressure_respect_to_microphone.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>You should make the below a function so that you are not typing in constant repeatedly. That is prone to error.</p>

<pre class="ql-syntax" spellcheck="false">mic_sensitivity_dBVperPa = -38
gain_setting_in_dB = -5.7
dBperPa = 94
dist = 20.0
RLper20uPa_at_dist = RL_dB[:,np.where(np.isclose(r,dist,atol=0.01))[0]]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/duty-cycle-investigation/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>